### ISSUE TITLE
Installation

### ISSUE THREAD

**Issue by miriamposner** [UCSCLibrary/GettySuggest#10]
I'm wondering if I installed this correctly. Does the whole file go into the plugins file in Omeka, or just a part of it? (Using Omeka Classic v2.6.)
<img width="571" alt="screen shot 2018-04-16 at 2 28 59 pm" src="https://user-images.githubusercontent.com/889407/38836328-91f02eba-4182-11e8-88c9-fcb17347d126.png">

**Comment by NedHenry** [comment]
Thanks for reporting this issue. If you download the zip file for the
plugin, I think you have to extract the folder from inside the zip file and
then move that folder into the Omeka plugins directory. Is that what you're
doing already? If so, let us know exactly what the file or folders you put
into the plugin directory is called, and maybe we can figure out what's
going wrong for you.
Cheers!
Ned

**Comment by miriamposner** [comment]
Yup, that's what I did! The file is called GettySuggest-master and it's inside the plugins folder. Thanks!

**Comment by NedHenry** [comment]
Try renaming the folder to remove the '-master' part. Also make sure the
plug-in files are right inside that folder.

**Comment by miriamposner** [comment]
Yes, renaming it worked! Thanks a lot!

**Comment by NedHenry** [comment]
No worries! These naming conventions confused me a ton when I started with
Omeka :)
